include/pybind11/numpy.h

```diff
@@ -561,35 +595,36 @@ class array : public buffer {
     }
 };
 
-template <typename T, int ExtraFlags = array::forcecast> class array_t : public array {
+template <typename T, int ExtraFlags = array<>::forcecast> class array_t : public array<> {
```

Comment:
I though that since `array_t` is the C++ class for numpy array, we would want to always check the access to its elements. But I can propagate the policy parameter to it if this looks better.